v1.1/badge/open-badge-v2.0.descriptor.json

```diff
@@ -92,32 +92,14 @@
       }
     },
     {
-      "label": "Alignment",
-      "path": [
```

Comment:
this is okay to remove these paths?

Reply:
We want to dispaly only the alignment URL, not the name and the framework. This change applies to all schemas.